player/tests/test.rs

```diff
@@ -144,7 +144,9 @@ impl Test<'_> {
                 }
             };
 
-            assert_eq!(&expected_data[..], contents);
+            if &expected_data[..] != contents {
+                panic!("Test expectation is not met!");
```

Comment:
Could we use `assert_eq!(&expected_data[..], contents, "Test expectation is not met!");` here?

Reply:
I think that would still try to print out the comparison to the console? We need to either compare less data, or introduce some metric to output (i.e. percent of difference). In any way, the current state of things is not the best